akka/src/test/scala/anorm/AkkaStreamSpec.scala

```diff
@@ -31,6 +26,18 @@ class AkkaStreamSpec extends org.specs2.mutable.Specification {
       }
     }
 
+    "be done if the stream run through" in { implicit ee: EE =>
+      @volatile var applied = false
+      withQueryResult(stringList :+ "A" :+ "B" :+ "C") { implicit con =>
+        Await.result(AkkaStream.source(SQL"SELECT * FROM Test", SqlParser.scalar[String]).
```

Comment:
actually that doesn't work since I want the materialized value and not the return value of the sink.